agent/rpc_to.go

```diff
@@ -76,6 +76,59 @@ func (a *orbAgent) sendGroupMembershipRequest() error {
 	return nil
 }
 
+func (a *orbAgent) askOrbTags() error {
+	defer func() {
+		if a.tagsRequestTicker == nil {
+			a.tagsRequestTicker = time.NewTicker(retryRequestFixedTime * retryRequestDuration)
+		}
+		var ctx context.Context
+		ctx, a.groupRequestSucceeded = a.extendContext("retryAskOrbTags")
+		go func(ctx context.Context) {
```

Comment:
wait - do I understand correctly that this will restart the entire agent every ticker? we can't do that. why do we need a ticker? the control plane should send new agent config only when it changes

Reply:
Actually the restart is when it successfully receives the Tags, and we flag it as a success, stopping the ticker and the retry mechanism.